src/templates/generic-story/generic-story.tsx

```diff
@@ -44,18 +44,22 @@ const Wrapper = styled.div`
  */
 export const GenericStory = ({ story, config }: CommonTemplateTypes) => {
   const footerText = get(config, ["publisherConfig", "publisher-settings", "copyright"], null);
+  let showPoweredByQt = get(config, ["opts", "featureConfig", "showPoweredByQt"], true);
   const infiniteScrollInlineConfig = get(
     config,
     ["opts", "featureConfig", "infiniteScroll", "infiniteScrollInlineConfig"],
     null
   );
-  let lastComponent = <Footer text={footerText} />;
+
+  showPoweredByQt = typeof showPoweredByQt === "function" ? showPoweredByQt() : showPoweredByQt;
```

Comment:
if `showPoweredByQt` is a function, wont it need parameters such as config to consume?